kitsune/users/migrations/0029_remove_profile_is_fxa_migrated.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.9 on 2024-01-23 05:58
+
+from django.db import migrations
+
+def purge_is_fxa_migrated_false_users(apps, schema_editor):
+    Profile = apps.get_model("users", "Profile")
+    Profile.objects.filter(is_fxa_migrated=False).delete()
+    
+class Migration(migrations.Migration):
+    dependencies = [
+        ("users", "0028_alter_profile_bio_and_upper_name_idx"),
+    ]
+
+    operations = [
+        migrations.RunPython(purge_is_fxa_migrated_false_users),
```

Comment:
the data migration needs a `noop` backwards method in order to be possible to go backwards if we ever need to